pom.xml

```diff
@@ -57,6 +57,7 @@
         <jackson.version>2.9.10</jackson.version>
         <jackson.bom.version>2.9.10.20200621</jackson.bom.version>
 
+        <guava.version>28.1-jre</guava.version>
```

Comment:
What is the version in parent poms? Can we make this change in the common pom file, so all the project can inherit the right version?

Reply:
This is the pom.xml in Common.

Reply:
Arrggg..... stupid me :)